lib/assets/stylesheets/modules/_search.scss

```diff
@@ -68,6 +105,7 @@ html.has-search-results-open {
   }
 .search-results__inner {
   position: relative;
+  height: 100%;
```

Comment:
Can't see what this line does, can you explain?

Reply:
I think it's the same as [here](https://github.com/alphagov/tech-docs-gem/pull/209#discussion_r589415139). I've removed this line.